webpages/settings/style.css

```diff
@@ -6,7 +6,7 @@ body {
   background: #222;
   font-family: "Sora", sans-serif;
   margin: 0;
-  height: 100vh;
+  min-height: 100vh;
```

Comment:
This change appears to cause styling issues when the addon container doesn't overflow:

![image](https://user-images.githubusercontent.com/17484114/99882640-cadc7600-2c00-11eb-87c1-97d6662baa92.png)